common/autoware_universe_utils/test/src/geometry/test_geometry.cpp

```diff
@@ -1980,3 +1983,1053 @@ TEST(geometry, intersectPolygonRand)
       (gjk_no_intersect_ns + gjk_intersect_ns) / 1e6);
   }
 }
+
+TEST(geometry, area)
+{
+  using autoware::universe_utils::area;
+  using autoware::universe_utils::alt::ConvexPolygon2d;
+  using autoware::universe_utils::alt::Point2d;
+
+  {  // Clockwise
+    const Point2d p1 = {0.0, 0.0};
+    const Point2d p2 = {0.0, 7.0};
+    const Point2d p3 = {4.0, 2.0};
+    const Point2d p4 = {2.0, 0.0};
+    const auto result = area(ConvexPolygon2d({p1, p2, p3, p4}));
+
+    EXPECT_NEAR(result, 16.0, epsilon);
+  }
+
+  {  // Counter-clockwise
+    const Point2d p1 = {0.0, 0.0};
+    const Point2d p2 = {2.0, 0.0};
+    const Point2d p3 = {4.0, 2.0};
+    const Point2d p4 = {0.0, 7.0};
+    const auto result = area(ConvexPolygon2d({p1, p2, p3, p4}));
+
+    EXPECT_NEAR(result, -16.0, epsilon);
+  }
+}
+
+TEST(geometry, convexHull)
+{
+  using autoware::universe_utils::convex_hull;
+  using autoware::universe_utils::alt::Points2d;
+
+  {
+    Points2d points;
+    points.push_back({2.0, 1.3});
+    points.push_back({2.4, 1.7});
+    points.push_back({2.8, 1.8});
+    points.push_back({3.4, 1.2});
+    points.push_back({3.7, 1.6});
+    points.push_back({3.4, 2.0});
+    points.push_back({4.1, 3.0});
+    points.push_back({5.3, 2.6});
+    points.push_back({5.4, 1.2});
+    points.push_back({4.9, 0.8});
+    points.push_back({2.9, 0.7});
+    const auto result = convex_hull(points);
+
+    EXPECT_EQ(result.vertices().size(), 7);
```

Comment:
(nitpick)
With `EXPECT_...` the test will not stop even if the condition is false. In that case this can lead to a segfault in the following lines (e.g., if the size is 0).
In order to prevent tests from crashing, such condition on vector size should be checked with `ASSERT_EQ`.
```suggestion
    ASSERT_EQ(result.vertices().size(), 7);
```

Reply:
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/7778/commits/1e825e247538b5a5327f2f547f8ecff8a21524db